### DISCUSSION TITLE
Not Watched overlay disappeared.

### DISCUSSION THREAD

**Discussion by sabonim38** [meisnate12/Plex-Meta-Manager#1983] [Q&A] [answered]
So I had to run my config.yml with remove overlay: true. After that all Plex Not Watched overlay (Orange triangle in top right corner) has disappeared. Their status is still set to Not Watched in Metadata, but the triangle is not just showing.

Anyone have an idear why?

I have set remove overlay: false again and have tried to run it again.

I had to set it to true, because PMM didn't change the overlay to the new settings, I had put in...

Any help will be appreciated. 

Edit: If I change the movies to "watched" the green check mark appears. But If I change it back still no orange triangle.

**Comment by mikenobbs** [comment, marked as answer]
PMM doesn't control that aspect of Plex, it's part of the UI. Plex changed it relatively recently, rather than an orange triangle for unplayed media you now get a green tick on media you have watched,. You can't have both, it's either or depending on the client 🙂

**Reply by sabonim38** [reply]
Thanks - because I had the triangle yesterday. I know because some of my language overlay in the top was covered by it ;-)
